storey/sql_driver.py

```diff
@@ -115,15 +115,23 @@ def _get_where_clause(self, key, sql_table):
             if i != 0:
                 where_clause += " and "
             if sql_table.columns[self._primary_key[i]].type.python_type == str:
-                where_clause += f'{self._primary_key[i]}="{key[i]}"'
+                where_clause += (
+                    f'{sql_table.name}.[{self._primary_key[i]}]="{key[i]}"'
```

Comment:
Good, please note that if you use alias - you must use it's new name later on :)

Reply:
I am using it's new name no?

Reply:
yea all good:)